docs/api-documentation/rpc.md

```diff
@@ -116,5 +155,36 @@ Note, this is Tendermint-like compatibility, that should be refactored.
 
 `tx(hash: bytes)`: queries status of the transaction by hash, returns FinalTransactionResult that includes status, logs and result: `{"status": "Completed", "logs": [{"hash": "<hash>", "lines": [], "receipts": [], "result": null}]}`
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx params:=["<base 58 of transaction hash>"] id=dontcare`
+`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx id=dontcare 'params:=["<base 58 of transaction hash>"]'`
 
+```javascript
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": {
+        "status": "Completed",
+        "transactions": [
```

Comment:
@ilblackdragon Is this reasonable? Having `transactions` as a response to `tx` request looks odd to me. Should it be `actions`?

Reply:
@evgenykuzyakov discussed in Discord, can paste the response from there.

Reply:
@evgenykuzyakov This looks much cleaner!

Reply:
@evgenykuzyakov I have noticed that you use `TransactionResultView` for the `receipts` as well... Shouldn't we create a separate View for receipt?